### DISCUSSION TITLE
Should we get our theme to comply with WP repo standard and submit to the repository?

### DISCUSSION THREAD

**Discussion by AmnestyAM** [amnestywebsite/humanity-theme#320] [Feature Requests]
add our theme to the WordPress repository 
https://learn.wordpress.org/lesson-plan/upload-a-theme-to-the-wordpress-repository/ 
https://make.wordpress.org/themes/handbook/review/required/ 

No custom post types, so we need to move the pop in to plugin and sidebars

**Comment by mikachan** [comment]
👋 Here's a check-list from the results of the [Theme Check plugin](https://en-gb.wordpress.org/plugins/theme-check/):

### Required:
- [ ] The theme is missing a readme.txt file ([Example](https://github.com/WordPress/twentytwentyfour/blob/trunk/readme.txt)). This should include license info for any assets that are included with the theme, such as fonts and images.
- [ ] The theme's text domain (`amnesty`) does not match the theme's slug (`humanity-theme`). It's important to keep this consistent in the translations functions.
- [ ] The theme's screenshot.png should be a screenshot of the homepage of the theme. It can use example content.
- [ ] The filter `user_contactmethods` should be moved to a plugin.
- [ ] The filter `mime_types` should be moved to a plugin.
- [ ] Any uses of `register_taxonomy` should be moved to a plugin.
- [ ] Any uses of `register_post_type` should be moved to a plugin.
- [ ] Any uses of `register_block_type` should be moved to a plugin.
- [ ] Any uses of `add_shortcode` should be moved to a plugin.
- [ ] All the logic in `wp-head-cleanup.php` should be moved to a plugin.
- [ ] `base64_encode` is not allowed - this logic could probably be moved to a plugin too.

### Warnings:
- [ ] The theme's screenshot.png is 1.5 MB, it would be best to compress this as much as possible.
- [ ] All translation functions should include the correct text domain for the theme as the second argument.
